x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/rules/api/retry.ts

```diff
@@ -65,7 +65,10 @@ export const registerSiemRuleMigrationsRetryRoute = (
               ],
             };
 
-            const filters: RuleMigrationFilters = { failed, notFullyTranslated };
+            const filters: RuleMigrationFilters = {
+              failed,
```

Comment:
Shouldn't we also check if `failed` is _undefined_?

Reply:
Oh, I see we later check if `=== true` or `=== false` explicitly. Okay, but then let's treat `fullyTranslated` the same way: always set the key, but it's possible to be _undefined_:

```ts
              fullyTranslated: notFullyTranslated != null ? !notFullyTranslated : undefined,
```

Reply:
And, ideally, the API parameter should also be called `fullyTranslated`

Reply:
done in https://github.com/elastic/kibana/pull/206089/commits/46b31ada2e4a49140b8a1fd2566a6a3463badbb5